translate/README.md

```diff
@@ -1,22 +1,58 @@
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Translate API Ruby Samples
+# Google Cloud Translation API Ruby Samples
 
-
-With the [Google Translate API][translate_docs], you can dynamically translate
-text between thousands of language pairs.
+The [Google Cloud Translation API][translate_docs] can dynamically translate
+text between thousands of language pairs. The Cloud Translation API lets
+websites and programs integrate with the translation service programmatically.
+The Google Translation API is part of the larger Cloud Machine Learning API
```

Comment:
I think the "Cloud Machine Learning API family" language is a bit outdated. I would remove this sentence.